pixart.js

```diff
@@ -1 +1,43 @@
+console.log('script.js loaded...');
+//globals
+var pixelForm = document.getElementById('form');
+var colorBox = document.querySelector('.brush');
+var colorInput = document.getElementById('color-field');
+var colorButton = document.getElementById('set-color');
 
+//create 20 div's with class .square and append them to document.body
+for (var i = 2; i < 8000; i++) {
+  var extraDiv = document.createElement('div');
+  extraDiv.className = 'square'
```

Comment:
I really like the way you broke down your code here instead of chaining multiple methods on top of each other! I think abstracting `document.createElement("div")` into `var extraDiv` is a great way to accomplish this, and makes it much easier to understand/read over :+1: